.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseSetParser.cs

```diff
@@ -190,20 +223,143 @@ private DateTimeResolutionResult ParserTimeEveryday(string text, DateObject refD
             var ret = new DateTimeResolutionResult();
 
             var ers = this.config.TimeExtractor.Extract(text, refDate);
+            var ers1 = this.config.TimePeriodExtractor.Extract(text, refDate);
+
+            if (ers.Count == 1)
```

Comment:
Which cases now fall here?

Reply:
It will parse the value under tasksmode for the input text, such as daily morning.